src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1988,7 +1988,7 @@ First_select_entries_to_clean_up.=First_select_entries_to_clean_up.
 Cleanup_entry=Cleanup_entry
 Autogenerate_PDF_Names=Autogenerate_PDF_Names
 Auto-generating_PDF-Names_does_not_support_undo._Continue?=Auto-generating_PDF-Names_does_not_support_undo._Continue?
-You_have_toggled_the_BibLaTeX_mode.=You_have_toggled_the_BibLaTeX_mode.
+You_have_toggled_an_edit_mode_switch.=You_have_toggled_an_edit_mode_switch.
```

Comment:
Why not "You have toggled_the_%0_mode" which can be used with "BibLaTeX" and "BibTeX" ?